pkg/crawler/crawler.go

```diff
@@ -41,6 +41,7 @@ type Option struct {
 
 func NewCrawler(opt Option) Crawler {
 	client := retryablehttp.NewClient()
+	client.RetryMax = 50
```

Comment:
50 might be too much. How much is the default value? I think we want to double the default value.

Reply:
Default value is 5.
Set 10 in https://github.com/aquasecurity/trivy-java-db/pull/26/commits/41e14cb0070e31f70251d1a6d389cfe9a5445a00.